python/equistore-operations/equistore/operations/_utils.py

```diff
@@ -192,3 +275,90 @@ def _check_gradient_presence(block: TensorBlock, parameters: List[str], fname: s
                 f"requested gradient '{parameter}' in {fname} is not defined "
                 "in this tensor"
             )
+
+
+def _labels_equal(a: Labels, b: Labels, exact_order: bool) -> bool:
+    """
+    For 2 :py:class:`Labels` objects ``a`` and ``b``, returns true if they are
+    exactly equivalent in names, values, and elemental positions. Assumes that
+    the Labels are already searchable, i.e. they belong to a parent TensorBlock
+    or TensorMap.
+    """
+    # They can only be equivalent if the same length
+    if len(a) != len(b):
+        return False
+    # Check the names
+    if not np.all(a.names == b.names):
+        return False
+    if exact_order:
+        return np.all(np.array(a == b))
+    return np.all([a_i in b for a_i in a])
```

Comment:
We have to remove all numpy operations here for torch script right? But maybe we do this in an upcoming PR....

Reply:
yes we need to dispatch them all

Reply:
okay!

Reply:
Is this function still needed or useful for some future PR? I could not find a usage in this PR

Reply:
I think this function is a relic of the past; I wrote it ages ago to help with metadata checks when you couldn't just do `labels_a == labels_b`. So fairly sure it's not needed anymore, but not 100 %. @Luthaf ?

Reply:
Then it was removed already and was somehow reintroduced on a rebase here? Would just remove it

Reply:
Okay I will remove it!